utils/realtek-poe/Makefile

```diff
@@ -3,14 +3,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=realtek-poe
-PKG_RELEASE:=2
+PKG_RELEASE:=1
 PKG_LICENSE:=GPL-2.0
```

Comment:
This is deprecated SPDX License Identifier and it needs to be fixed.
See: https://spdx.org/licenses/